ibm_catalog.json

```diff
@@ -0,0 +1,110 @@
+{
+  "products": [
+    {
+      "name": "terraform-ibm-cloudant",
+      "label": "IBM Cloudant module",
+      "product_kind": "solution",
+      "tags": [
+        "ibm_created",
+        "target_terraform",
+        "terraform",
+        "reference_architecture",
+        "solution"
+      ],
+      "keywords": [
+        "cloudant",
+        "IaC",
+        "infrastructure as code",
+        "terraform",
+        "solution",
+        "cloudant dedicated",
+        "cloud dedicated"
+      ],
+      "short_description": "Creates an IBM Cloudant Dedicated instance in IBM Cloud Dedicated environment.",
+      "long_description": "This architecture creates an IBM Cloudant Dedicated instance in IBM Cloud Dedicated environment and supports provisioning the following resources:\n- A resource group, if one is not passed in.\n- An IBM Cloudant Dedicated instance on IBM Cloud Dedicated\n- A IBM Cloudant database\n",
```

Comment:
Again, this needs to be higher level since its the offering level, not flavour level description. Maybe go with:

"This architecture supports creating and configuring an IBM Cloudant instance and database"